webpack/components/Search/Search.js

```diff
@@ -38,74 +39,55 @@ class Search extends Component {
       params.params || {},
     ];
 
-    if (autoCompleteParams[0] !== '') {
+    if (params.endpoint !== '') {
```

Comment:
What if it's undefined?

Reply:
Touché

Reply:
Question mark added.

Reply:
If `params` exists but doesn't have an `endpoint` property, `params?.endpoint` would evaluate to `undefined` and we'd be comparing `undefined !== ''`, which is true.  I assume that's not what you want to happen.  I feel like a simple
```js
if (params?.endpoint) {
```
is a better check?

Reply:
After examination of the output of getAutoCompleteParams, this entire block seems pointless.
Rewrote using your suggestion/destructuring.